src/wdio/services/TerraToolkitServeStaticService.js

```diff
@@ -0,0 +1,44 @@
+import serve from '../../../scripts/serve/serve-static';
+
+export default class TerraToolkitServeStaticService {
+  async onPrepare(config) {
+    if (!config.webpackConfig) {
+      // eslint-disable-next-line no-console
+      console.log('[Terra-Toolkit:serve-static] No webpack configuration provided');
+      return;
+    }
+
+    const webpackConfig = config.webpackConfig;
+    const port = ((config || {}).serveStatic || {}).port || 8080;
```

Comment:
`(config || {})` need this?

Reply:
not if i default config to be an empty object.